android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/UploadFileContract.java

```diff
@@ -0,0 +1,22 @@
+package org.sagebionetworks.bridge.android.manager;
+
+import android.provider.BaseColumns;
+
+/**
+ * Created by jameskizer on 5/26/17.
+ */
+
+public class UploadFileContract {
+
+    private UploadFileContract() {}
+
+    public static class UploadFileSchema implements BaseColumns {
+
+        public static final String TABLE_NAME = "upload_file";
```

Comment:
Since we use GSON do we need to specifically define these?  Seems like extra maintenance.